services/core-api/app/api/users/minespace/resources/minespace_user.py

```
@@ -42,6 +42,22 @@ def post(self):
             new_mum.save()
         return new_user
 
+    @api.doc(description='Update an existing Minespace Users mine list')
+    @api.marshal_with(MINESPACE_USER_MODEL)
+    @requires_role_mine_admin
+    def put(self, user_id):
+        contact = MinespaceUser.find_by_id(user_id)
+        if not contact:
+            raise NotFound('Contact not found.')
+        data = self.parser.parse_args()
+        current_app.logger.debug('TESTING IT HIT THIS API')
+        mines = data.mine_guids
+        result = MinespaceUser.update_minelist(user_id, mines)
+        if result:
+            contact.save()
+            return contact
+        else:
+            print("Error: Cannot update user ${user_id}'s mines")
```

To ensure the client handles an error correctly raise an InternalServerError with that error message rather than a console print. That error is found in the werkzeug.exceptions namespace.